TextToSpeech/impl/TTSManager.cpp

```diff
@@ -114,9 +114,33 @@ TTS_Error TTSManager::listVoices(std::string language, std::vector<std::string>
     return TTS_OK;
 }
 
+TTS_Error TTSManager::listLocalVoices(std::string language, std::vector<std::string> &voices) {
+     
+     std::string key = std::string("voice_for_");
+     if(language.empty())
+     {
+        voices.push_back(m_defaultConfiguration.localvoice());
+     }
+     else
+     {
+         key += language;
+         if(m_defaultConfiguration.m_others_local.find(key) != m_defaultConfiguration.m_others_local.end())
```

Comment:
you can use the result of the search to avoid another one in case value was found. i.e.:

```
auto it = m_defaultConfiguration.m_others_local.find(key);
if (it != m_defaultConfiguration.m_others_local.end()) 
{
  voices.push_back(it->second);
}
```